.github/workflows/docker.yaml

```diff
@@ -28,7 +29,8 @@ jobs:
   docker:
     needs: pre_job
     if: |
-      needs.pre_job.outputs.should_skip != 'true' &&
+      (needs.pre_job.outputs.should_skip != 'true' ||
```

Comment:
just for consistency, may want a space before "needs." and after the parenthesis here.